### PR TITLE
refactor: Change defaults in usecases

### PR DESCRIPTION
**Pull request by dentremor**
Closes https://github.com/alphahorizonio/consumat.io/issues/154

### CODE REVIEW

File: consumatio/external/db/db.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def media_data_exists(self: object, user_id: int, media: str,
             logger.info("media_data entry exists in database")
             return True
 
-    def media_Data(self: object, user_id: int, media: str, media_id: int,
+    def media_data(self: object, user_id: int, media: str, media_id: int,
```
Comment (JakWai01) [inline consumatio/external/db/db.py:138]: Please also change the code where the function is called. Like this, the application crashes

**Review by JakWai01 — CHANGES_REQUESTED** [review]
Please take a look at the suggested changes. Also, could be maybe refuse to accept changes to set the value of favorite or numberOfWatchedEpisodes to null, so we won't ever have null as a value in the database. This would create inconsistencies in the data model.

**Comment by dentremor** [comment]
The functionality you ask for would conflict with the following closed Issue: https://github.com/alphahorizonio/consumat.io/issues/126

It should also be possible to set the numberOfWatchedEpisodes to null, since this value cannot be set sensibly for all media types (e.g. movie).

**Review by JakWai01 — APPROVED** [review]
LGTM

**Pull request merged**